gallery/src/components/NxDropdown/NxDropdownLinksExample.tsx

```diff
@@ -17,15 +17,15 @@ function NxDropdownActionsExample() {
                 className="extra-class"
                 isOpen={isOpen}
                 onToggleCollapse={onToggleCollapse}>
-      <a href="#/" className="nx-dropdown-link disabled">
+      <a href="https://www.google.com/" className="nx-dropdown-link disabled">
         <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
         <span>Go somewhere</span>
       </a>
-      <a href="#/" className="nx-dropdown-link">
+      <a href="https://www.sonatype.com/" className="nx-dropdown-link">
         <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
         <span>Go somewhere else</span>
       </a>
-      <a href="#/" className="nx-dropdown-link">
+      <a href="https://developer.mozilla.org/en-US/docs/Web" className="nx-dropdown-link">
```

Comment:
Why these changes? `#/` is valid, it's our home page

Reply:
> Why these changes? `#/` is valid, it's our home page

If you look at `/#/pages/Dropdown`, at the `Links Example`, each link has the external link icon to the left of it, so I changed the links from the internal `#/` to different external links.

Reply:
> If you look at /#/pages/Dropdown, at the Links Example, each link has the external link icon to the left of it, so I changed the links from the internal #/ to different external links.

Hmm so they do.  But the CSS classes aren't set up to imply that these links are specifically external, _all_ dropdown menu links have this icon.  Which makes me wonder if that's supposed to be the case.  Checking Kaleido, I don't see mockups for dropdown menu links at all.  I think we need to talk to the designers about that.  For now though I guess your change is fine